carbon-design-system/carbon-website
# feat(ContributionGallery): now in MDX!

closes https://github.com/carbon-design-system/carbon-website/issues/3718

### What's changed

- [Shadowed](https://gatsby-theme-carbon.vercel.app/guides/shadowing) the `ArticleCard` component to change the token values to reflect our latest Tile guidance ♿ .
- Updated Contributing > Get Started with a Contribution Gallery that repurposes the ArticleCard.

### Testing

- Check the Contributing > Get Started page. Make sure the gallery looks right.
- Notice that our `ArticleCards` everywhere on the site will be changed to reflect new Tile guidance.
